fem/ceed/coefficient.hpp

```diff
@@ -82,20 +82,20 @@ void InitCoefficient(mfem::Coefficient *Q, mfem::Mesh &mesh,
                      const mfem::IntegrationRule &ir,
                      Coefficient*& coeff_ptr, Context &ctx)
 {
+   ConstantCoefficient *coeff;
```

Comment:
This is later assigned to `GridFunctionCoefficient*`, it's probably better to keep the declarations inside the `if` statement, and just give them unique names, `const_coeff`, `gf_coeff`, etc.

Reply:
yes, yes

I somehow missed the difference

Reply:
Hopefully https://github.com/mfem/mfem/pull/2839/commits/247de1cf04ed70bbe4183e7a898fd74d9b14d4fb is OK